bin/node-template/Cargo.toml

```diff
@@ -36,6 +35,7 @@ sc-client = { version = "2.0.0", path = "../../client/" }
 node-template-runtime = { version = "2.0.0", path = "runtime" }
 sp-runtime = { version = "2.0.0", path = "../../primitives/runtime" }
 sc-basic-authority = { path  = "../../client/basic-authorship" }
+structopt = "=0.3.8"
```

Comment:
Very good point :grin: 

I was thinking: don't you think it would be best that the users use structopt from a public export in `sc_cli` like structopt does for clap?

I mean:
```rust
pub structopt::StructOpt;
```

Reply:
Yeah we can also re-export, that is up to you :)